### ISSUE TITLE
Unable to import modules - please add ESM fields to package.json

### ISSUE THREAD

**Issue by rondonjon** [reactivemarkets/react-financial-charts#520]
#### I'm submitting a...

- [x] bug
- [ ] feature
- [ ] chore

#### What is the current behavior

The published npm modules cannot be imported.

#### What is the expected behavior

Packages should be published CJS- and/or ESM-compliant.

#### Please tell us about your environment

Version: 1.2.2

Typescript compiler options:

```
"allowJs": false,
"allowSyntheticDefaultImports": true,
"declaration": true,
"esModuleInterop": true,
"forceConsistentCasingInFileNames": true,
"isolatedModules": true,
"jsx": "preserve"
"lib": ["dom", "dom.iterable", "esnext"],
"module": "esnext",
"moduleResolution": "node",
"noEmit": true,
"noFallthroughCasesInSwitch": true
"noImplicitReturns": true,
"noUnusedLocals": true,
"noUnusedParameters": true,
"resolveJsonModule": true,
"skipLibCheck": true,
"strict": true,
"strictNullChecks": true,
"target": "es5",
"types": ["node", "jest"],
"typeRoots": ["./node_modules/@types"],
```

Import statement:

```
import {
  ...
  ZoomButtons,
  withDeviceRatio,
  withSize,
} from "react-financial-charts"
```

Error:

```
export * from "@react-financial-charts/annotations";
^^^^^^

SyntaxError: Unexpected token 'export'
```

#### Other information

Could this be caused by missing ESM fields in the `package.json` of the published `react-financial-chart` modules? 

I understand that the modules are published with ESM exports, and this is in fact quite common today and supported by most bundlers, but only if it goes along with some hint like `"type": "module"` or `"module": "./lib/index.js"` in `package.json`. Without further annotation, most bundlers (or in my case: TSC) will try to process the `"main"` export as CommonJS + ES5 and fail when they encounter the ESM-specific `export` keyword.

This problem seems to apply to more (or possibly all) modules of this project; I tried to import those in `@react-financial-charts` directly and got the same error:

```
export * from "./Annotate";
^^^^^^

SyntaxError: Unexpected token 'export'
```

**Comment by markmcdowell** [comment]
We can the type property, that's not a problem. I don't think the module property ever made it past proposal.

**Comment by rondonjon** [comment]
Sounds good, @markmcdowell , thanks in advance :+1:

**Comment by markmcdowell** [comment]
Released in [v1.3.0](https://github.com/reactivemarkets/react-financial-charts/releases/tag/v1.3.0)
